core/modules/installer/installer.info

```diff
@@ -0,0 +1,7 @@
+name = Installer
```

Comment:
I think we should call this "Project Installer" as Installer on it's own  is not self-explanitory.  This will jive nicely with https://github.com/backdrop/backdrop-issues/issues/1757 too.